### PR TITLE
Make clear that element retrieval waits for no-one

### PR DESCRIPTION
**Pull request by shs96c**
Closes issue #438.

---
This change is [Reviewable](https://reviewable.io/reviews/w3c/webdriver/489)

### CODE REVIEW

File: webdriver-spec.html [hunk 1]
```diff
@@ -3919,7 +3919,10 @@
  allow lookup of individual elements and collections of elements.
  Element retrieval searches are performed
  using pre-order traversal of the document’s nodes
- that match the provided selector’s expression.
+ that match the provided selector’s expression, and ignore the value
+ of the <a href="#dfn-page-load-strategy" class="internalDFN"
```
Comment (AutomatedTester) [inline webdriver-spec.html]: this just needs to be `<a>page load strategy</a>`.

File: webdriver-spec.html [hunk 2]
```diff
@@ -3919,7 +3919,10 @@
  allow lookup of individual elements and collections of elements.
  Element retrieval searches are performed
  using pre-order traversal of the document’s nodes
- that match the provided selector’s expression.
+ that match the provided selector’s expression, and ignore the value
```
Comment (AutomatedTester) [inline webdriver-spec.html]: Why does this need to be mentioned? Find Elements is not, as currently written, influenced by Page load strategy.

File: webdriver-spec.html (hunk 1 above)
Comment (andreastt) [inline webdriver-spec.html]: Ignoring the value of the page load strategy makes no sense in this context. The page load strategy only applies when, well, actually loading a document.

File: webdriver-spec.html (hunk 2 above)
Comment (andreastt) [inline webdriver-spec.html]: +1

Comment (shs96c) [inline webdriver-spec.html]: Because of issue #438, which is why I linked to it in the description.

File: webdriver-spec.html (hunk 1 above)
Comment (shs96c) [inline webdriver-spec.html]: See #438. If someone found this unclear, and it's not obfuscating things to clarify, it seems wise to help clear things up.

Comment (andreastt) [inline webdriver-spec.html]: I did see #438, which is about _implicit wait_ strategy not being mentioned in element retrieval. The element retrieval algorithm doesn’t explain what the driver should do when implicitly waiting for elements to appear in the DOM.

**Comment by shs96c** [comment]
Thanks to @andreastt and @AutomatedTester for gently pointing out that it's the implicit wait strategy that should be used.

**Pull request closed without merging**